src/rust/python-parser/test_import_parsing.c++

```diff
@@ -1,25 +1,7 @@
-// Copyright (c) 2017-2022 Cloudflare, Inc.
-// Licensed under the Apache 2.0 license found in the LICENSE file or at:
-//     https://opensource.org/licenses/Apache-2.0
+#include "import_parsing.h"
```

Comment:
test file name conventions are <module>-test.c++
So this should be `python-parser-test.c++`